documentation/staging/content/samples/azure-kubernetes-service/troubleshooting.md

```diff
@@ -171,7 +171,7 @@ Fri Nov 27 13:07:14 CST 2020
 You may run into a timeout while installing the operator and get the following error:
 
 ```shell
-$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+$ helm install weblogic-operator weblogic-operator/weblogic-operator \
```

Comment:
@edburns You may want to review this change to see if it is correct for Azure. The future "4.0" operator doc is recommending using a repot to get the helm chart instead of a file throughout, but I don't know if Azure has a different need.